src/index.js

```diff
@@ -1,14 +1,50 @@
+/* eslint-disable no-unused-vars */
 import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const todoItems = [];
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+function renderTodo(todo) {
+  const list = document.querySelector('.js-todo-list');
 
-  return element;
+  const isChecked = todo.checked ? 'done' : '';
+  const node = document.createElement('li');
+  node.setAttribute('class', `todo-item ${isChecked} border flex height pad-left`);
+  node.setAttribute('data-key', todo.index);
+  node.innerHTML = `
+    <div class="left flex">
```

Comment:
- [ ] As described in the project requirements, please manage some default tasks to be rendered and sorted on page load by the index number, and not the Time. Please see the next comment about that matter 🙏🏻😊

Reply:
* done 👍

Reply:
* done 👍